src/main/java/fr/xephi/authme/api/v3/AuthMeApi.java

```diff
@@ -133,20 +133,17 @@ public Location getLastLocation(Player player) {
     }
 
     /**
-     * Get the registration ip address of a player.
+     * Returns the AuthMe info of the given player's name, or null if the player doesn't exist.
      *
-     * @param playerName The name of the player to process
-     * @return The registration ip address of the player
+     * @param playerName The player name to look up
+     * @return AuthMe player info, or null if not available
      */
-    public String getRegistrationIp(String playerName) {
+    public AuthMePlayer getPlayerInfo(String playerName) {
```

Comment:
Api v4, deprecate api v3

Reply:
Optional

Reply:
> Api v4, deprecate api v3

No.
This is just a method addition and shouldn't warrant a new API version. Unless you mean the removed method, but that's one that's two days old and wasn't part of any release (even beta).

Reply:
Fair, but i would like to change that in the future, with proper versioning, maybe by splitting the api into a separate maven module.